server/controllers/refer/referralsController.test.ts

```diff
@@ -164,4 +164,19 @@ describe('ReferralsController', () => {
       })
     })
   })
+
+  describe('update', () => {
+    it('asks the service to update a referral and redirects to the show action', async () => {
+      const referral = referralFactory.build({ oasysConfirmed: false })
+      referralService.getReferral.mockResolvedValue(referral)
+
+      request.body.oasysConfirmed = true
+
+      const requestHandler = referralsController.update()
+      await requestHandler(request, response, next)
+
+      expect(response.redirect).toHaveBeenCalledWith(referPaths.show({ referralId: referral.id }))
+      expect(referralService.updateReferral).toHaveBeenCalledWith(token, referral.id, { oasysConfirmed: true })
+    })
```

Comment:
Add something like ``describe('updating `oasysConfirmed`')`` to nest this, so we can add another test for `reason` within this outer `describe` block later